src/mcts/search.cc

```diff
@@ -137,7 +137,7 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
     const auto wl = edge.GetWL();
     const auto d = edge.GetD();
     const int w = static_cast<int>(std::round(500.0 * (1.0 + wl - d)));
-    const auto q = edge.GetQ(default_q, draw_score);
+    const auto q = edge.GetQ(default_q, draw_score, false);
```

Comment:
Can we add a comment here to make clear what is passed here?
```c++
const auto q = edge.GetQ(default_q, draw_score, /* logit_q= */ false);
```

Reply:
Done.

Reply:
Not really done. :)

Reply:
Heh I got all the other ones but missed this one? Nice.
Fixed.